engine/src/main/resources/application-base.yml

```diff
@@ -111,4 +111,14 @@ callback:
       backoff-policy:
         initial-interval-ms: 5000
         max-interval-ms: 30000
-        multiplier: 2.0
\ No newline at end of file
+        multiplier: 2.0
+
+docker:
+  registry:
+    url:
```

Comment:
What is the point to add empty values here?

Reply:
I removed the values for pushing the code, I believe a placeholder could work here but the code is prepared to handle these null values.

Reply:
This doesn't answer the question. Any authentication values should come from outside.
What you put in base config should be "meaningful default". I don't see how empty values are meaningful